README.md

```diff
@@ -19,13 +19,20 @@ from thermal_base import utils
 image = ThermalImage(image_path="path/to/image", camera_manufacturer="dji/flir")
 ```
 
-**Disclaimer**
-
-Tested against data from: 
-* Flir encoding format: DJI XT-2, Flir E-40 (Temperature values go out of the image frame if the image height is <255 pixels)
-* DJI-encoding format: DJI Zenmuse H20-T
-* Thermapp (available in past commits)
+## Supported formats
```

Comment:
In the `Usage` section, can you also add a mention that this repo can be used in conjunction with Thermal-Image-Analysis repo for an interactive experience

Reply:
And similarly, in the other repo, can you mention that it relies on this repo for a backend?